src/theory/strings/theory_strings_preprocess.cpp

```diff
@@ -54,7 +54,7 @@ Node StringsPreprocess::reduce(Node t,
   Trace("strings-preprocess-debug")
       << "StringsPreprocess::reduce: " << t << std::endl;
   Node retNode = t;
-  NodeManager* nm = NodeManager::currentNM();
+  NodeManager* nm = t.getNodeManager();
```

Comment:
Use `nodeManager()`.

Reply:
Same issue here. It is a static function.